tests/keras/backend/reference_operations.py

```diff
@@ -493,6 +493,20 @@ def variable(value, dtype=None, name=None, constraint=None):
     return np.array(value, dtype)
 
 
+def dropout(x, level, noise_shape=None, seed=None):
+    if noise_shape is None:
+        noise_shape = x.shape
+    if learning_phase():
+        np.random.seed(seed)
```

Comment:
The meaning of `seed` in the API is to make the dropout *node* in the graph deterministic, but not constant. In this implementation the op would be constant if passed a seed argument. Not super important for just debugging use cases, but easy to fix.

I suggest you simply remove the `np.random.seed(seed)` statement, this would be closer to the intended behavior.

Reply:
Sure thing.